### PR TITLE
[lensfun] Add new port

### PR DESCRIPTION
**Pull request by JackBoosY**
- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [x] The name of the port matches an existing name for this component on https://repology.org/ if possible, and/or is strongly associated with that component on search engines.
- [x] Optional dependencies are resolved in exactly one way. For example, if the component is built with CMake, all `find_package` calls are REQUIRED, are satisfied by `vcpkg.json`'s declared dependencies, or disabled with [CMAKE_DISABLE_FIND_PACKAGE_Xxx](https://cmake.org/cmake/help/latest/variable/CMAKE_DISABLE_FIND_PACKAGE_PackageName.html)
- [x] The versioning scheme in `vcpkg.json` matches what upstream says.
- [x] The license declaration in `vcpkg.json` matches what upstream says.
- [x] The installed as the "copyright" file matches what upstream says.
- [x] The source code of the component installed comes from an authoritative source.
~~- [ ] The generated "usage text" is accurate. See [adding-usage](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/examples/adding-usage.md) for context.~~
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is in the new port's versions file.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

File: ports/lensfun/portfile.cmake [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    "${CURRENT_PACKAGES_DIR}/debug/include"
+    "${CURRENT_PACKAGES_DIR}/debug/share"
+)
+
+file(INSTALL "${SOURCE_PATH}/docs/gpl-3.0.txt"
+    DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}"
+    RENAME copyright)
+
+file(INSTALL "${SOURCE_PATH}/docs/lgpl-3.0.txt"
+    DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}")
```
Comment (LilyWangLL) [inline ports/lensfun/portfile.cmake:59]:
```suggestion
vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/docs/gpl-3.0.txt" "${SOURCE_PATH}/docs/lgpl-3.0.txt")
```

**Review by LilyWangLL — CHANGES_REQUESTED** [review]

**Comment by LilyWangLL** [comment]
`lensfun[sse2,tool]:x64-windows` build failed with the following error:
```
F:\lens\buildtrees\lensfun\src\v0.3.4-2231d3b854.clean\apps\lenstool\rgbpixel.h(54): note: see declaration of 'RGBpixel'
F:\lens\buildtrees\lensfun\src\v0.3.4-2231d3b854.clean\apps\lenstool\image.cpp(17): warning C4244: 'return': conversion from 'double' to 'float', possible loss of data
F:\lens\buildtrees\lensfun\src\v0.3.4-2231d3b854.clean\apps\lenstool\image.cpp(22): warning C4244: 'return': conversion from 'double' to 'float', possible loss of data
F:\lens\buildtrees\lensfun\src\v0.3.4-2231d3b854.clean\apps\lenstool\image.cpp(42): warning C4273: 'lanczos_func': inconsistent dll linkage
F:\lens\buildtrees\lensfun\src\v0.3.4-2231d3b854.clean\apps\lenstool\image.h(45): note: see previous definition of 'private: static float * Image::lanczos_func'
F:\lens\buildtrees\lensfun\src\v0.3.4-2231d3b854.clean\apps\lenstool\image.cpp(42): error C2491: 'Image::lanczos_func': definition of dllimport static data member not allowed
```

**Comment by JackBoosY** [comment]
Remove feature `tool` since it always need dynamic build and have issues in OSX.

**Comment by LilyWangLL** [comment]
All features passed with following triplets:
```
x86-windows
x64-windows
x64-windows-static
```

**Review by LilyWangLL — APPROVED** [review]

File: ports/lensfun/vcpkg.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    }
+  ],
+  "features": {
+    "sse": {
+      "description": "Build with support for SSE"
+    },
+    "sse2": {
+      "description": "Build with support for SSE2"
+    }
+  }
```
Comment (Neumann-A) [inline ports/lensfun/vcpkg.json:26]: @JackBoosY Please remove these features. Arch settings are not features and SSE2 should be support on everything !arm.

**Pull request merged**